docs/navbar.html

```diff
@@ -1,5 +1,6 @@
 <div class="navbar">
   <div id="nav-left" class="nav-left"></div>
+  <div id="nav-title" class="nav-title"><h1 class="nav-header" id="nav-title-header"></h1></div>
```

Comment:
I don't think this id nav-title is used anywhere (mods made to class nav-title and id nav-title-header)